packages/connect-popup/src/view/common.tsx

```diff
@@ -140,7 +136,7 @@ export const initMessageChannel = async (
 
             // POPUP.HANDSHAKE successfully received back from the iframe
             if (await broadcastHandshake) {
-                setState({ broadcast, systemInfo, iframe });
+                setState({ broadcast, systemInfo });
```

Comment:
I am not sure of the reason of adding this `iframe` here but maybe we can achieve it in a different way, because it looks like if we `getIframeElement` before there are some events not triggering in web extensions.

Reply:
<img width="371" alt="image" src="https://github.com/trezor/trezor-suite/assets/30367552/48e07049-cb98-4c9d-923a-97f710d3f26d">

I made those changes in order to make webusb work. If you try it on develop, it is possible to pair webusb device in popup. If you try it after your changes in this PR, button does not react. So we need to go deeper to fix both